src/assets/seed/data-structures/BinarySearchTree.js

```diff
@@ -126,11 +126,13 @@ class BinarySearchTree {
 			// node only has right child
 			else if (!node.left) {
 				node.value = node.right.value
+				node.left = node.right.right
```

Comment:
Hi @mansisce, so sorry for the extremely late reply. It looks like this line is supposed to be:

```
node.left = node.right.left
```

If you get around to fixing this I can merge!